ThemeDecorator/ThemeDecorator.module.less

```diff
@@ -46,12 +46,6 @@
 	.disabled({
 		.vendor-opacity(@agate-disabled-opacity);
 
-		// Focused and disabled, must restore their natural opacity, then assign the faded text/content color
-		.focus({
-			.vendor-opacity(1);
-			color: @agate-spotlight-disabled-text-color;
```

Comment:
I am careful because this modification can affect all skins and component. actually, Silicon skin is affected and I cannot distinguish the disable status and disable with focus status.
I suggest some alternatives. Please let me know your opinion

1. Apply some style to Dropdown from Item.
There is no issue in the sampler. Because the Dropdown children is Item component.
Can't we just bring the Item component style into the DropDown style ?

2. Change the @agate-spotlight-disabled-text-color of Gallium.
current:   fade(@agate-spotlight-color, 40%); and @agate-spotlight-color is white.

3. Keep the opacity change.

		.focus({
			.vendor-opacity(1); // keep and change this value (option)
			// color: @agate-spotlight-disabled-text-color; // remove
		});

And, If you want revmoe " color: @agate-spotlight-disabled-text-color; "@agate-spotlight-disabled-text-color" is not used anymore.Please remove it